service/s3/s3manager/download_test.go

```diff
@@ -39,7 +39,7 @@ func dlLoggingSvc(data []byte) (*s3.S3, *[]string, *[]string) {
 		fin, _ := strconv.ParseInt(rng[2], 10, 64)
 		fin++
 
-		if fin > int64(len(data)) {
+		if fin >= int64(len(data)) {
```

Comment:
the extra `=` here isn't really needed since `fin` will be set back to the same value.